playwright-e2e/pages/cancer-selection.ts

```diff
@@ -0,0 +1,6 @@
+interface CancerSelection {
+    cancerSearch: string;
+    expectedCancerResult:string;
+    time: string; // could be age at diagnosis, year of diagnosis, etc.
+    numTimesToHitDownArrow:number; // how many times to press the down key to get from search hit to the result
```

Comment:
True.  These reference values should change very rarely.  And with the initial hint to narrow the search, it should reduce the surface area of flakiness even more.

Reply:
As for the `time` field, it's deliberately vague because in some studies, this is age at diagnosis, in others its the year of diagnosis.  How about `diagnosisAt`?

Reply:
Okay. I understand why now. `time` is fine without change.